src/promptflow/tests/sdk_cli_azure_test/recording_utilities/utils.py

```diff
@@ -202,7 +202,7 @@ def sanitize_pfs_body(body: str) -> str:
 
 
 def sanitize_email(value: str) -> str:
-    return re.sub(r"([\w\.-]+)@([\w\.-]+)", r"{}@\2".format(SanitizedValues.EMAIL_USERNAME), value)
+    return re.sub(r"([\w\.-]+)@(microsoft.com)", r"{}@\2".format(SanitizedValues.EMAIL_USERNAME), value)
```

Comment:
copy from online 
regex = /^[A-Za-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[A-Za-z0-9!#$%&'*+/=?^_`{|}~-]+)*@([A-Za-z0-9]+(?:-[A-Za-z0-9]+)?\.)+[A-Za-z0-9]+(?:-[A-Za-z0-9]+)?$/;

Reply:
https://www.cnblogs.com/itman70s/p/regex.html

Reply:
good catch, let me try this regex

Reply:
offline synced, this comment targets to refine username part; opened a work item to track internally.